tcr-engine/toolchain/gradle.go

```diff
@@ -30,7 +30,7 @@ func init() {
 				Os:        GetAllOsNames(),
 				Arch:      GetAllArchNames(),
 				Path:      "gradle",
-				Arguments: []string{"build", "-x", "test"},
```

Comment:
What were the arguments for storing default toolchain values in go rather than in a resource file? (Can resource files be embedded in the executable like in Java?)

Reply:
There were not really arguments. When I started writing TCR that was the most straightforward approach to have these in Go code directly. Go has a mechanism to have filesystems embedded in the binary (thus providing a mechanism similar to resource files). This being said I'm not sure it would provide much benefit as we would have to deliver a new version of TCR every time we tune these "resource" files.

Reply:
Maybe we could use the same file we use in examples?

Reply:
We could indeed. But don't you think this goes far beyond the scope of this pull request?

Reply:
Created issue #178 to cover this proposal